ivy/functional/backends/jax/activations.py

```diff
@@ -46,6 +46,24 @@ def sigmoid(
     return 1 / (1 + jnp.exp(-x))
 
 
+def segment_max(
```

Comment:
Hi, is there any reason `jax.ops.segment_max` was not used?

Reply:
I thought that this could work but now I have added it